uber/models.py

```diff
@@ -615,8 +615,8 @@ def change_badge(self, attendee, badge_type, badge_num=None):
             return 'Badge updated'
 
         def everyone(self):
```

Comment:
I'm not sure that ``everyone`` is even used anywhere since it doesn't have good performance, but I'm glad we're keeping it up to date so long as it exists.